CGPs/0017.md

```diff
@@ -0,0 +1,112 @@
+# CGP [0017]: Activation of Celo Community Fund
+
+- Date: 2020-12-01
+- Author(s): @devme25 @patricknbaron @DeeM297
+- Status: DRAFT
+- Governance Proposal ID #: [if submitted]
+- Date Executed: [if executed]
```

Comment:
Add another item called "Discussions To:" with a url to a forum post for discussions on this CGP